lib/fieldhand/options.rb

```diff
@@ -36,6 +37,15 @@ def bearer_token
       options[:bearer_token]
     end
 
+    # Build custom headers
+    def headers
+      headers = {}
+      headers.merge!(options[:headers]) if options[:headers]
```

Comment:
Could we simplify this with `Hash#fetch`?

```ruby
headers = options.fetch(:headers, {})
headers['Authorization'] = "Bearer #{bearer_token}" if bearer_token

headers
```

Then, as it uses `bearer_token`, we'd want to move that method definition _below_ this one to keep it [pyramidal](http://scottmatthewman.github.io/2015/11/09/hack-like-a-journalist-slides.html)?